app/views/CalculationAdditionalView.scala.html

```diff
@@ -35,9 +35,11 @@
 
 @layout(pageTitle = messages("calculation.title.additional"), isPageFullWidth = false, isUserAgent = user.isAgent,
     extraBreadcrumbs = Some(Seq((messages("index.breadcrumb.index"), controllers.routes.IndexController.onPageLoad().url)))) {
-        <span class="govuk-caption-l">@messages("calculation.caption", startDate, endDate)</span>
-       <h1 class="govuk-heading-l">
-        @messages("calculation.heading.additional")</h1>
+     <header class="hmrc-page-heading govuk-!-margin-top-0">
+         <h1 class="govuk-heading-xl govuk-!-margin-bottom-5"> @messages("calculation.heading.additional")</h1>
+         <p class="govuk-caption-xl hmrc-caption-xl">
+            <span class="govuk-visually-hidden">@messages("calculation.heading.additional.hiddenText")</span>@messages("calculation.caption", startDate, endDate)</p>
```

Comment:
You will probably need to add a space after the hidden text to prevent it saying:
The period dates are1 January (which may confuse a screen reader)
```suggestion
            <span class="govuk-visually-hidden">@messages("calculation.heading.additional.hiddenText") 
</span>@messages("calculation.caption", startDate, endDate)</p>
```

Reply:
Now I'm thinking it may not affect screen readers but it may clean up the test below